commands/create_cluster.go

```diff
@@ -237,36 +240,37 @@ func createClusterExecutionOutput(cmd *cobra.Command, args []string) {
 
 	result, err := addCluster(aca)
 	if err != nil {
-		handleCommonErrors(err)
+		errors.HandleCommonErrors(err)
+		client.HandleErrors(err)
```

Comment:
Could these be moved inside the switch statement?

Reply:
I wouldn't know how. They are meant to print a message in case a known error happened and exit.